### PR TITLE
EWPP-1777: Enable 'Include past revisions' option in composite fields.

### PR DESCRIPTION
**Pull request by yenyasinn**

### CODE REVIEW

**Review by nagyad — DISMISSED** [review]

File: modules/oe_content_event/oe_content_event.post_update.php [hunk 1]
```diff
@@ -133,3 +133,20 @@ function oe_content_event_post_update_20003(): void {
...
+
+/**
+ * Set "composite revisions" option for reference fields.
+ */
+function oe_content_event_post_update_20004(): void {
+  $fields = [
+    'node.oe_event.oe_event_contact' => TRUE,
+    'node.oe_event.oe_event_media' => FALSE,
+    'node.oe_event.oe_event_venue' => TRUE,
+    'oe_event_speaker.oe_default.oe_person' => FALSE,
```
Comment (22Alexandra) [inline modules/oe_content_event/oe_content_event.post_update.php]: This config belongs to the submodule providing the EventSpeaker subentity, so I think it should be updated in a post_update of the submodule. The submodule is not a dependency of the main module so the config might not event exist in some cases.

File: modules/oe_content_organisation/oe_content_organisation.post_update.php [hunk 2]
```diff
@@ -95,3 +95,20 @@ function oe_content_organisation_post_update_20006() {
...
+
+/**
+ * Set "composite revisions" option for reference fields.
+ */
+function oe_content_organisation_post_update_20007(): void {
+  $fields = [
+    'node.oe_organisation.oe_organisation_chart' => FALSE,
+    'node.oe_organisation.oe_organisation_contact' => TRUE,
+    'node.oe_organisation.oe_organisation_logo' => FALSE,
+    'oe_contact.oe_organisation_reference.oe_node_reference' => FALSE,
```
Comment (22Alexandra) [inline modules/oe_content_organisation/oe_content_organisation.post_update.php]: The same remark as for the EventSpeaker field config.

File: modules/oe_content_organisation/modules/oe_content_organisation_person_reference/config/install/field.field.node.oe_organisation.oe_organisation_persons.yml [hunk 3]
```diff
@@ -10,6 +10,7 @@ dependencies:
 third_party_settings:
   composite_reference:
     composite: false
+    composite_revisions: false
```
Comment (22Alexandra) [inline modules/oe_content_organisation/modules/oe_content_organisation_person_reference/config/install/field.field.node.oe_organisation.oe_organisation_persons.yml:13]: This field config should be updated in a post_update too.

File: modules/oe_content_publication/oe_content_publication.post_update.php [hunk 4]
```diff
@@ -209,3 +209,23 @@ function oe_content_publication_post_update_20002(): void {
...
 }
+
+/**
+ * Set "composite revisions" option for reference fields.
+ */
+function oe_content_publication_post_update_20003(): void {
+  $field_config = FieldConfig::load('node.oe_publication.oe_publication_contacts');
+  $field_config->setThirdPartySetting('composite_reference', 'composite_revisions', TRUE);
+  $field_config->save();
+
```
Comment (22Alexandra) [inline modules/oe_content_publication/oe_content_publication.post_update.php]: This is updated in the loop.
```suggestion
```

**Review by 22Alexandra — CHANGES_REQUESTED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Review by nagyad — APPROVED** [review]

**Pull request merged**